data/hazards.txt

```diff
@@ -124,3 +124,20 @@ hazard "Rulei Gravity Field"
 
 effect "gravity well"
 	"sound" "gravity well"
+
+
+hazard "Core Sheltered Heat"
+	"duration" 1
+	"strength" 0.65 0.85
+	"range" 50000
+	weapon
+		"burn damage" 0.06
```

Comment:
The burn is explicitly intentional.

Reply:
It is a slightly abstract approximation for the effect of radio-thermal spalling (thermal spalling triggered by the presence of X-rays, and other high-energy radiation types) which *does* have a persistent effect that takes a bit to wear off.

And yes, it is a small effect on this hazard, it's a larger effect on the other, more severe, hazard which is exclusive to the Xapleaux PR (because it's not used in any of these systems).

Reply:
> Are you referring to cosmic ray induced spallation of nuclei?

No.

It is mechanically different because if you are approaching the limit, you might then jump out of that system, and be temporarily disabled on the other end of the jump.
It applies a small risk to jumping from one bad situation into another if you don't think about where you're going properly, which is an intentional part of the hazard.
This is a novel gameplay experience that is not currently simulated elsewhere.

Which leads me to reprise a statement I've made a few times: Just because something is not _necessary_ is not intrinsically an argument against it.

Reply:
Having a minor transition into relevantly dangerous burn damage is a far better way to introduce both the concept and the risk it poses.

If these values are too low to do that, then the proper response is not to nix them, but to instead change the reaction in these introductory systems to more obviously do so.

Reply:
FYI Sheratan has the `Core Sheltered Heat` hazard. But I like that, it suggests there's more to the area. I don't want to get involved in which systems have what hazards though, I'm not fussed about whether Sheratan has the hazard or not.

Reply:
(low burn damage does not mean it boils off quickly, it's exponential decay and dissipates at the same rate at all times- the amount present is cut in half approximately every 69 frames. It'll have a low impact, but it'll still linger for the exact same amount of time as any other amount of burn damage)